sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,7 @@
+select "genres"."name" as "genre",
+       count("filmId") as "Appearing Lisa Monroe"
+  from "castMembers"
+  join "filmGenre" using ("filmId")
+  join "genres" using ("genreId")
+  where "actorId" = 178
```

Comment:
For the purpose of this exercise, only start with the information you know ahead of time:

---

![image](https://user-images.githubusercontent.com/16143771/161663118-abd2ff6b-7330-4676-ad42-831ef94160ed.png)

Reply:
![Screenshot 2022-04-04 192757](https://user-images.githubusercontent.com/25161542/161667085-2bd36db5-a84a-482b-80b2-19000fa33fbd.png)